oneview_redfish_toolkit/api/event.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import os
+
+from oneview_redfish_toolkit.api.redfish_json_validator \
+    import RedfishJsonValidator
+
+
+class Event(RedfishJsonValidator):
+    """Creates a Event Redfish dict
+
+        Populates self.redfish with some hardcoded Event
+        values and with the response from OneView.
+    """
+
+    SCHEMA_NAME = 'Event'
+
+    def __init__(self, oneview_alert, event_type):
```

Comment:
There are some info needed in the event that is stored in the subscription object... and we will need to update it on this event before delivering to each client, right?

Reply:
i don't know what you refer to. Please let me know which fields and where to get it.

Reply:
"Context" on "EventRecord" is defined as: "A context can be supplied at subscription time. This property is the context value supplied by the subscriber."

Maybe we'll have more from this kind...